lib/internal/errors.js

```diff
@@ -23,6 +23,7 @@ const overrideStackTrace = new WeakMap();
 const prepareStackTrace = (globalThis, error, trace) => {
   // API for node internals to override error stack formatting
   // without interfering with userland code.
+  console.log('dd',globalThis, error, trace);
```

Comment:
Was this added for debugging?
```suggestion
```

Reply:
yes, i'm added this for debugging remove this line